recipes/glib/all/conandata.yml

```diff
@@ -1,28 +1,28 @@
 sources:
   "2.65.0":
-    url: "https://download.gnome.org/sources/glib/2.65/glib-2.65.0.tar.xz"
-    sha256: "b041e63cd0ac1fccb486374022ade040d907aad29b278e27d9e43e9294a6e7a3"
+    url: "https://gitlab.gnome.org/GNOME/glib/-/archive/2.65.0/glib-2.65.0.tar.gz"
```

Comment:
It's not the official download location :s What happen with previous one? It works on my computer.

Reply:
Have a look at all the previous failures: every time I get a timeout for all the versions that are using that location. It works locally in my machine as well 🤷

Reply:
What about mirrors instead? It's possible to use a list for url and hashs

Reply:
I would have used the _mirror approach_, but the sha256 is different for the files in GitLab